src/Profile.cpp

```diff
@@ -155,7 +155,7 @@ namespace geopm
             m_is_enabled = true;
         }
         catch (const Exception &ex) {
-            if (m_process < 0) {
+            if (m_process >= 0) {
```

Comment:
m_process is set to a the result of comm_rank() in init_proc_comm() on line 140.  This means that m_process will always be non-negative if we enter the try statement.  I'm not sure why we would condition the warning message based on m_process being positive.